desktop/core/src/desktop/lib/raz/raz_client.py

```diff
@@ -144,17 +142,64 @@ def check_access(self, method, url, params=None, headers=None, data=None):
   def _handle_raz_req(self, raz_url, request_headers, request_data):
     if self.auth_type == 'kerberos':
       auth_handler = requests_kerberos.HTTPKerberosAuth(mutual_authentication=requests_kerberos.OPTIONAL)
-      raz_req = requests.post(raz_url, headers=request_headers, json=request_data, auth=auth_handler, verify=False)
+      raz_response = self._handle_raz_ha(raz_url, headers=request_headers, data=request_data, auth_handler=auth_handler)
+
     elif self.auth_type == 'jwt':
       jwt_token = fetch_jwt()
-
       if jwt_token is None:
         raise PopupException('Knox JWT is not available to send to RAZ.')
 
       request_headers['Authorization'] = 'Bearer %s' % (jwt_token)
-      raz_req = requests.post(raz_url, headers=request_headers, json=request_data, verify=False)
+      raz_response = self._handle_raz_ha(raz_url, headers=request_headers, data=request_data)
+    
+    if not raz_response:
+      raise PopupException('Failed to receive response from RAZ server.')
+
+    return raz_response
+
+
+  def _handle_raz_ha(self, raz_urls, headers, data, auth_handler=None, verify=False):
+    raz_urls_list = raz_urls.split(',')
```

Comment:
is this backward compatible? if the configuration in the existing setup is not comma separated value, will this fail?

Reply:
Yes this is backward compatible. If only 1 Raz URL is set in the config, it will work like non-HA mode.

I've added a unit test case for this scenario.

Reply:
@Harshg999 , I believe we should also address a scenario where a user inputs something like "http://xyz,". In this case, the second element would be an empty one. Have we considered handling this particular case?